src/app/core/core.module.ts

```diff
@@ -2,18 +2,16 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { HeaderComponent } from './components/header/header.component';
 import { FooterComponent } from './components/footer/footer.component';
-import { BreadcrumbsComponent } from './components/breadcrumbs/breadcrumbs.component';
+import { BreadcrumbsComponent } from '../courses/components/breadcrumbs/breadcrumbs.component';
```

Comment:
Breadcrumbs should be in the core module because the component is used throughout the application.